Info.plist

```diff
@@ -50,6 +50,18 @@
 				<string>pocket-next</string>
 			</array>
 		</dict>
+		<dict>
+			<key>CFBundleTypeRole</key>
+			<string>Editor</string>
+			<key>CFBundleURLIconFile</key>
+			<string>Icon</string>
```

Comment:
do we need an icon here?

Reply:
I'm not sure what the icon is used for.

Reply:
just curious - do you have any resource guide to point me to learn more about setting this up? Why are we adding this to the `.plist`. Is this to support URL Scheme? 

I was researching since I'm not too familiar with URL scheme, but it seems it was used for deep linking in previous iOS versions. I read this [blog post](https://byby.dev/ios-deep-linking) - it seems that universal links has a couple of benefits over custom URL schemes. 

Does this also suppose universal links?

Reply:
Adding a URL Type tells the app what URL Schemes it can handle.

In this case we are registering `pocket://` as a scheme that we want to handle. 
My understanding is that when this app is installed iOS will be told that if it recieves a `pocket://something/something` URL iOS knows to power up Pocket and hand off that URL to Pocket for handling. We parse out the intent and navigate to the related content in code.

It's been a while, but iirc Universal Links don't use this. Universal Links are standard `https://` links. You set up an endpoint on your server to tell Apple that this URL can also be opened by your app.

https://facebook.com/apple-app-site-association
Is Facebook's Universal Link association file, for example. That means that you can try to open `https://www.facebook.com/someContent` on either the Facebook app, if installed, or just in a browser if not.

Reply:
It seems that we don't need this at all